tests/StandortControllerTest.cs

```diff
@@ -8,13 +11,17 @@ namespace EWS;
 [TestClass]
 public class StandortControllerTest
 {
+    private readonly string testStandortBezeichnung = "Blue Jeans Pelati";
```

Comment:
PP: Da dieser Wert nie ändert, kann man daraus eine Konstante machen:
```suggestion
    private const string TestStandortBezeichnung = "Blue Jeans Pelati";
```